Manifest.toml

```diff
@@ -8,20 +8,18 @@ version = "1.0.1"
 
```

Comment:
Why do we track Manifest.toml files here?

Reply:
This is a [good thread](https://discourse.julialang.org/t/does-manifest-toml-belong-in-the-repository/12029) on when it should be tracked or not.

Reply:
Oh - makes sense. Thanks!